src/CalculatorUITestFramework/CalculatorResults.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using OpenQA.Selenium.Appium.Windows;
+
+namespace CalculatorUITestFramework
+{
+    public class CalculatorResults
+    {
+        private WindowsDriver<WindowsElement> session => WinAppDriver.Instance.CalculatorSession;
+        public WindowsElement CalculatorAlwaysOnTopResults => this.session.TryFindElementByAccessibilityId("CalculatorAlwaysOnTopResults");
```

Comment:
>public [](start = 8, length = 6)

make this private

Reply:
As part of commit a8c27208b04f206e706a029edd3930692db69383
In test framework, where it worked, I made elements private